### DISCUSSION TITLE
Custom Encounter Locations

### DISCUSSION THREAD

**Discussion by SterlingMoore** [jynnie/soullocke#17] [Ideas]
Hi, really enjoying the app, thank you for making it.
I wanted to propose the idea of having the ability to add a custom encounter, which could be used for Eggs and Static encounters in games. For instance, I'm using the app to track myself and a friend playing through Radical Red, which offers more options for encounters than the typical Kanto map, and I can't seem to find a way to add more encounters beyond the ones already entered into the app, so some encounters are left unaccounted for. 
Again, really love the app and appreciate the obvious care you've put into it to make it really appealing.

**Comment by jynnie** [comment]
That makes sense! I'll add it to my list for the next update! Thanks for using it!

**Comment by jynnie** [comment]
This will be in next weeks update!

![2023-02-25 17 10 13](https://user-images.githubusercontent.com/7284723/221387022-e2da224d-1db4-4fa0-86ee-095bbaeea460.gif)
